<launch>
	<arg name="node_start_delay" default="20.0" />
	<param name="force_x" type="double" value="0.05" />
	<param name="force_y" type="double" value="0.05" />
	<param name="force_z" type="double" value="0" />
	<param name="time_d" type="double" value="3" />
	<include file="$(find ur_gazebo)/launch/ur5.launch"/>
	<node pkg="joint_pub" name="joint_pub" type="ik_solver_2" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' "/>

</launch>
